src/scripts/main.js

```diff
@@ -1,3 +1,299 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = table.querySelector('tbody');
+const arr = Array.from(tableBody.rows);
+let count = 0;
+let target;
+
+// Active row
+table.addEventListener('click', e => {
+  const tr = e.target.closest('TR');
+  const currentActive = table.querySelector('tr.active');
+
+  if (currentActive) {
+    currentActive.classList.toggle('active');
+  }
+
+  tr.classList.toggle('active');
+});
+
+// Sort table
+table.addEventListener('click', e => {
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  if (target !== e.target) {
+    target = e.target;
+    count = 0;
+  } else {
+    count++;
+  }
+
+  if (count % 2) {
+    sortDESC(e.target.cellIndex);
+  } else {
+    sortASC(e.target.cellIndex);
+  }
+});
```

Comment:
Use boolean varaible for example isAsc = true by default and onClick you can toggle it